mysql/backup.pl

```diff
@@ -98,6 +98,7 @@
 				"root",
 				$config{'backup_single_'.$sf},
 				$config{'backup_quick_'.$sf},
+				0,
```

Comment:
@iliarostovtsev - there's another `0` here which should be `undef`

Reply:
Okay, thanks.